MericCelik/assignment1/LinkedListNode.java

```diff
@@ -0,0 +1,12 @@
+public class LinkedListNode {
+    Object element;
```

Comment:
optional: When writing this, did you notice that calls to get the value of the element will give you 'unchecked cast to X' warnings? A nice and elegant solution to this is to use Generics instead of holding the values as Object.

Reply:
It is my first time using generics in classes, I hope I did in the right way. Glad to learn, it is a great method!